Gradata/tests/test_embeddings_integration.py

```
@@ -26,22 +26,22 @@ def test_local_fallback_when_api_unavailable(self):
         client = EmbeddingClient(api_url=None)
-        with patch.object(client, '_embed_local', return_value=[0.1, 0.2, 0.3]):
+        with patch.object(client, "_embed_local", return_value=[0.1, 0.2, 0.3]):
             result = client.embed("test")
             assert result == [0.1, 0.2, 0.3]
 
     def test_api_preferred_over_local(self):
         client = EmbeddingClient(api_url="https://example.com/embed", api_token="tok")
-        with patch.object(client, '_embed_api', return_value=[0.5, 0.6]) as mock_api:
-            with patch.object(client, '_embed_local') as mock_local:
+        with patch.object(client, "_embed_api", return_value=[0.5, 0.6]) as mock_api:
+            with patch.object(client, "_embed_local") as mock_local:
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)